SharedPackages/BrowserServicesKit/Sources/NetworkProtection/KeyManagement/NetworkProtectionTokenStore.swift

```diff
@@ -167,47 +167,50 @@ public final class NetworkProtectionKeychainTokenStore: NetworkProtectionTokenSt
 /// Writing a new auth token will replace the old one.
 public final class NetworkProtectionKeychainTokenStoreV2: AuthTokenStoring {
     private let keychainStore: NetworkProtectionKeychainStore
-    private let errorEvents: EventMapping<NetworkProtectionError>?
+    private let errorPixelHandler: EventMapping<NetworkProtectionError>?
 
     public struct Defaults {
+        static let bundleID = Bundle.main.bundleIdentifier ?? "com.duckduckgo.networkprotection"
         static let tokenStoreEntryLabel = "DuckDuckGo Network Protection Auth Token Container"
-        public static let tokenStoreService = "com.duckduckgo.networkprotection.authTokenContainer"
-        static let tokenStoreName = "com.duckduckgo.networkprotection.tokenContainer"
+        public static let tokenStoreService = "\(bundleID).authTokenContainer"
+        static let tokenStoreName = "\(bundleID).tokenContainer"
     }
 
     /// - isSubscriptionEnabled: Controls whether the subscription access token is used to authenticate with the NetP backend
     /// - accessTokenProvider: Defines how to actually retrieve the subscription access token
     public init(keychainType: KeychainType,
                 serviceName: String = Defaults.tokenStoreService,
-                errorEvents: EventMapping<NetworkProtectionError>?
+                errorPixelHandler: EventMapping<NetworkProtectionError>?
```

Comment:
This is ok, but just adding a note... the mapping can be used for more than just pixels, and it's coincidental that we're only using this for pixels.  It's more of an `errorEventsHandler` IMO.

But again, no need to change this, just a though.

Reply:
renamed